Package@swift-6.0.swift

```diff
@@ -0,0 +1,109 @@
+// swift-tools-version:6.0
+
+import PackageDescription
+
+let package = Package(
+    name: "swift-aws-lambda-runtime",
+    platforms: [.macOS(.v15)],
+    products: [
+        // this library exports `AWSLambdaRuntimeCore` and adds Foundation convenience methods
+        .library(name: "AWSLambdaRuntime", targets: ["AWSLambdaRuntime"]),
+        // plugin to package the lambda, creating an archive that can be uploaded to AWS
+        // requires Linux or at least macOS v15
+        .plugin(name: "AWSLambdaPackager", targets: ["AWSLambdaPackager"]),
+        // for testing only
+        .library(name: "AWSLambdaTesting", targets: ["AWSLambdaTesting"]),
+    ],
+    traits: [
```

Comment:
Is this supported by the Swift 6.0 driver ?

Reply:
oversight. hupps.